config/routes.rb

```diff
@@ -14,15 +14,19 @@
   get "apps/show/:name(/:type(/:owner))" => "apps#show", as: "app", defaults: { type: "sys" }
   get "apps/icon/:name(/:type(/:owner))" => "apps#icon", as: "app_icon", defaults: { type: "sys" }
 
-  #FIXME: undo when ready to deploy app sharing to production, remove?
   if ENV['OOD_APP_SHARING'].present?
+    root "apps#index", defaults: { type: "usr" }
```

Comment:
If `OOD_APP_SHARING` is present and `OOD_APP_DEVELOPMENT` is not present, then we are setting the root to "apps#index" but "apps#index" route will not exist because that is defined in the `if ENV['OOD_APP_DEVELOPMENT'].present?` block.

Since we can still enable `OOD_APP_DEVELOPMENT` now without `OOD_APP_SHARING`, and since in AweSim we already have `OOD_APP_DEVELOPMENT=1`, for the sake of time, I can still pull this in. Then I'll open a separate issue to revisit configuration of app development and app sharing.

Reply:
Note this isn't an issue right now because we haven't advertised app sharing. But we did discuss "fixing" routes.rb so instead of `if ENV['OOD_APP_DEVELOPMENT'].present?` we would switch to using `if NavConfig.show_develop_dropdown` which would be replaced ultimately with a better feature flag like `AppConfig.developer_mode_enabled`. If and when we did that, the resulting code would crash the AweSim dashboard.